src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -176,8 +175,10 @@ class AddUserProfile extends Component {
                   </Col>
                   <Col md="6">
                     <FormGroup>
-                      <PhoneInput
-                        country={'us'}
+                    <PhoneInput
+                        country="US"
+                        enableLongNumbers = 'true'
```

Comment:
if we add this it will not restrict the length for countries.. like US need to allow 10 but now it is allowing 20..not showing validation also..

Reply:
So i just tried adding in numbers for both Canada and United States, and it did limit to 10 each. So I do think it is limiting based on region selected? I am not familiar with international numbers but if someone wanted to try one respective to a country/region where they're aware of phone number length they can give it a shot?

Reply:
I tested with India and united states..it worked as expected...looks good to me...